src/party_events.ts

```diff
@@ -76,3 +76,58 @@ export const getAllPartyEvents = async (fromBlock: number) => {
 
   return events;
 };
+
+export const getHaveSetNewPartyWithContributionCacheKey = (
+  partyAddress: string
+): string => `new_party_with_contribution_${partyAddress}`;
+
+export const haveSetNewPartyWithContribution = async (
```

Comment:
i feel like these fn names perhaps be a little more. clear, like `haveAlertedAboutNewParty`